packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -743,6 +771,15 @@ class PageView extends StatefulWidget {
   /// {@macro flutter.widgets.scrollable.dragStartBehavior}
   final DragStartBehavior dragStartBehavior;
 
+  /// Controls how scrolling navigation is handled for accessibility services.
+  ///
+  /// By default, accessibility traversal should navigate into the contents of
```

Comment:
The difference for a11y is: When this is on and a11y focus is at the end of the viewport and you move a11y to the next item it will scroll the list and move focus to the newly revealed item in the list. If it is off, a11y focus will move to the next item outside the scrollable.